consumer/types.go

```diff
@@ -774,3 +774,14 @@ func newFromBytes(d []byte, code int) (response, error) {
 		}, nil
 	}
 }
+
+type SMS struct {
+	MobileNo string `json:"mobileNo,omitempty"`
```

Comment:
Change the field name to `Mobile` and the marshall to just `mobile`. We try to keep fields as small as possible, and we use `type_face` for json objects (except for ebs compatibility)